x-pack/plugin/sql/qa/src/main/resources/agg.sql-spec

```diff
@@ -144,6 +144,10 @@ countDistinctAlias
 SELECT COUNT(DISTINCT hire_date) AS count FROM test_emp;
 countDistinctAndCountSimpleWithAlias
 SELECT COUNT(*) cnt, COUNT(DISTINCT first_name) as names, gender FROM test_emp GROUP BY gender ORDER BY gender;
+countWithFunctionFilterAndGroupBy
+SELECT COUNT(*) as c FROM test_emp WHERE ABS(salary) > 0 GROUP BY gender ORDER BY gender;
+countLiteralAndAliasWithFunctionFilterAndGroupBy
+SELECT 1 as t, gender as g, COUNT(*) as c FROM test_emp WHERE ABS(salary) > 0 GROUP BY g ORDER BY gender;
```

Comment:
Same here, please check more cases:

- aliased and unaliased literals
- more columns in group by (aliased and not aliased)
- more columns in the where